village/src/App.js

```diff
@@ -11,14 +12,45 @@ class App extends Component {
       smurfs: [],
     };
   }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:3333/smurfs')
+      .then(response => {
+        console.log ("this is the response:", response);
+        this.setState({smurfs: response.data }, () => console.log(this.state.smurfs));
+      })
+      .catch(error => {
+        console.error('Server Error', error);
+      });
+  }
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
   render() {
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+        {/* <SmurfForm />
+        <Smurfs smurfs={this.state.smurfs} /> */}
+
+        <header>
+          <h1>Welcome to the Smurf Village!</h1>
+          <h3>Join us in the Smurf Village here:</h3>
+          <Link to='/smurfs'>Smurf Village</Link>
+          <h3>Add a new member to our Smurf Village here:</h3>
+          <Link to='/smurfform'>New Smurf</Link>
+        </header>
+
+        <br></br>
```

Comment:
Using `<br>` like this isn't best practice. It's better to allow your CSS to create that space between those sections.